warggroundstation

```diff
@@ -0,0 +1 @@
+Subproject commit 684ad1dfd95a8025f05ba8f5afdead6c17c57f4d
```

Comment:
I think this file is related to git subtree, and should be version
controlled. If that is the case, then documentation should include
instructions or at last a reminder to use subtree. Now, should we be using
subtree at all? I an not sure.
On Mar 4, 2016 2:12 PM, "Serge Babayan" <notifications@github.com> wrote:

> In warggroundstation
> <https://github.com/UWARG/WARG-Ground-Station/pull/111#discussion_r55096669>
> :
>
> > @@ -0,0 +1 @@
> > +Subproject commit 684ad1dfd95a8025f05ba8f5afdead6c17c57f4d
>
> What is this file? Is it something generated by netbeans? Please delete it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UWARG/WARG-Ground-Station/pull/111/files#r55096669>.
>